src/styles/normalize.scss

```diff
@@ -0,0 +1,5 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: Roboto, Helvetica, sans-serif;
```

Comment:
Don't reset margins with * selector, reset for tags that really need it.
The main font family should be applied to the html or body element.